openapi/components/schemas/GatewayAccountConfig/LaCore.yaml

```
@@ -0,0 +1,28 @@
+description: LaCore gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: LaCore credentials object.
+        properties:
+          merchantId:
+            type: string
+            description: ID of the LaCore merchant.
+          username:
+            type: string
+            description: LaCore merchant username.
+          password:
+            type: string
+            description: LaCore merchant password.
```

suggestion: to avoid [noun stacking](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
            description: Username of the LaCore merchant.
          password:
            type: string
            description: Password of the LaCore merchant.
```